invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/tab.overrides

```diff
@@ -0,0 +1,34 @@
+/*******************************
+         Site Overrides
+*******************************/
+
+@import "@less/invenio_theme/theme/modules/tab.overrides";
+
+.ui.bottom.attached.segment.active.tab.borderless {
+  border: none !important;
+}
+
+.active.item.selected-menu-tab {
+  border-bottom: 2px solid #fb8273 !important;
+  border-left: none !important;
+  border-right: none !important;
+  border-top: none !important;
+  background: #F8F8F9 !important;
+  margin: 0em 0em -2px !important;
+}
+
+.ui.attached.tabular.border-thick-bottom.menu{
+  border-bottom: 2px solid rgba(34, 36, 38, 0.15);
+  margin-bottom: 10px;
+}
+
+.ui.attached.tabular.border-thick-bottom.menu{
```

Comment:
This last two selectors aim to display the grey background for the tabs menu. The tabs menu is placed inside a container with it's margins but we want the background to include all the width of the page. To achieve that some dirty workarounds with margin had to be used. Any feedback on how to achieve a cleaner solution is very appreciated.